data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+	passengers 2
+	to offer
+		has "event: saryd couple has money"
+	source "Shadow of Leaves"
+	destination "Far Garden"
+	on offer
+		conversation
+			`You're met with two familiar faces in the spaceport, Inturi and Aunaris, the Saryd couple you met some time ago, who were working on acquring the funding needed to build their own home.`
+			`	"Worked for a week in another ship, I did, until landed here we had," Inturi says, joking about his way of catching a ride on a ship without paying a transport fee.`
+			`	"Sleeping well, I have not been, but helped much with the campus administration, I have. A bonus, I was given," Aunaris tells you.`
+			`	They say they just need a transport to <destination>, the world they have chosen to put down roots, and after everything is done, they will work on getting you your payment.`
+			`	You show them to their bunks, and prepare for the trip.`
+				accept
+	on complete
+		"coalition jobs" ++
+
+
+
+mission "Saryd Couple 3"
+	name "Reversed House Spider"
+	description "Take Inturi to <destination>, where he will purchase a small Arach ship that he and Aunaris will repurpose into a home."
+	landing
+	passengers 1
+	to offer
+		has "Saryd Couple 2: done"
+	source "Far Garden"
+	destination "Saros"
+	on offer
+		conversation
+			`The couple leaves your ship, and head to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties here, in the most valued (and most sought-after) locations, and that they should still have enough money to pay you afterwards.`
+			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing, while Aunaris comes talk to you.`
+			`	"Kept watch of one property, for months we had. On a mountain village, overlooking a lake, and with access to a nearby forest, it is. A few days ago, purchased it was," she sighs. "Available, another plot of land neighboring it is, but over half our funds, the land alone costs."`
+			`	Inturi then looks over, as if he had just received a shock, and says, "That's it! A ship! Live in a ship, we could!"`
+			choice
+				`	"What do you mean?"`
+				`	"I like the enthusiasm, but unfortunately I'm not looking for any more crewmembers right now."`
+				`	"If you say so. Just don't forget the oranges, scurvy probably is a problem for Saryds too."`
+			`	"Live in a ship?" Aunaris repeats the sentence, puzzled.`
+			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`
```

Comment:
```suggestion
			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did. Often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy. Save time and money otherwise spent building a home, it would."`
```
Both your sentences would be run-on if rearranged back into normal structure.